src/scales/scale.logarithmic.js

```diff
@@ -52,294 +52,300 @@ function generateTicks(generationOptions, dataRange) {
 	return ticks;
 }
 
+// INTERNAL: static default options, registered in src/chart.js
```

Comment:
This comment is reduntant and can be removed

Reply:
Good catch, wasn't an easy one, especially on mobile.